libevmcore/EVMSchedule.h

```diff
@@ -108,6 +109,7 @@ static const EVMSchedule EIP158Schedule = []
 static const EVMSchedule MetropolisSchedule = []
 {
 	EVMSchedule schedule = EIP158Schedule;
+	schedule.blockhashGas = 800;
```

Comment:
It's uncertain to me if `800` should be applied from the beginning of `Metropolis` from 256 blocks into `Metropolis`.  I asked a question https://github.com/ethereum/EIPs/pull/210/files#r117211219